src/views/Award/Awards.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import Loading from '../../components/atoms/Loading/Loading.tsx';
+import { useAwards } from '../../api/Award/useAwards.ts';
+import AwardsContainer from './AwardsContainer.tsx';
+
+const Awards = () => {
+  const { id } = useParams();
+  const { awards, isAwardsLoading, isAwardsError } = useAwards(id);
+
+  if (isAwardsLoading)
+    return (
+      <AwardsContainer>
+        <Loading />
+      </AwardsContainer>
+    );
+
+  if (isAwardsError)
+    return (
+      <AwardsContainer>
+        <div>
+          Wystąpił błąd podczas wczytywania danych, proszę odświeżyć stronę.
+        </div>
+      </AwardsContainer>
+    );
+
+  return (
+    <AwardsContainer>
+      <div>
+        {awards?.map((award) => (
+          <li key={award.id}>
```

Comment:
`<li>` elements should be alwayes wrapped by `<ol>` or `<ul>` HTML element.
You could also just pass the `awards` list as a prop to the `<AwardsContainer>` and then render the list inside the component's code.

Reply:
Good point, I implemented this into the code!

Reply:
If there is no awards, we're getting `<ol><div> Brak nagród. </div></ol>`, which's not perfect but for me it's good enough.

Great job!